modules/osdk-cli-reference-scorecard.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-scorecard_{context}"]
+= scorecard
+
+The `operator-sdk scorecard` command runs the scorecard tool. The command
+takes one argument, either a bundle image or directory containing manifests and metadata. If the argument holds an image tag, the image must be present remotely.
+
+.`scorecard` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`-c`, `--config` (string)
+|Path to scorecard configuration file. The default is `bundle/tests/scorecard/config.yaml`.
+
+|`-h`, `--help`
+|Help output for the `scorecard` subcommand
+
+|`--kubeconfig` (string)
+|Path to `kubeconfig` file
+
+|`-L`, `--list`
+|Option to enable listing which tests are run
+
+|`-n`, --namespace (string)
+|Namespace in which to run the test images
+
+|`-o`, `--output` (string)
+|Output format for results. Available values are `text`, which is the default, and `json`.
+
+|`-l`, `--selector` (string)
+|Label selector to determine which tests are run
```

Comment:
What's the difference between this option and --list?

Reply:
Updated.